src/plonk.rs

```diff
@@ -15,6 +15,15 @@ pub const AUX_OFFSET: usize = 1;
 const SETUP_MIN_POW2: u32 = 20;
 const SETUP_MAX_POW2: u32 = 26;
 
+pub fn gen_key_monomial_form<E: Engine>(power: u32) -> Result<Crs<E, CrsForMonomialForm>, anyhow::Error> {
+    anyhow::ensure!(
+        (SETUP_MIN_POW2..=SETUP_MAX_POW2).contains(&power),
```

Comment:
is SETUP_MIN_POW2 still needed to be 20?

Reply:
refer to my README update:

> You may also want to manually edit and lower down `plonk::SETUP_MIN_POW2` in the codes to fast generate a relatively small-size SRS.

Reply:
Otherwise if we remove the check here, we will also need to remove the corresponding check in `prepare_setup_for_prover`. However, this seems less desired -- we still prefer to keep the range check in `prepare_setup_for_prover`.

Reply:
An alternative way is to have an ENV variable, or cli flag argument, to enable/disable those checks